lib/puppet/functions/deprecation.rb

```diff
@@ -8,15 +8,19 @@
   end
 
   def deprecation(key, message)
+    stacktrace = Puppet::Pops::PuppetStack.stacktrace()
```

Comment:
ugh, thanks for catching that. I know how that happened: https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/spec/functions/deprecation_spec.rb#L3 is a little too optimistic...

:-(

We'll retry tomorrow.